lib/sample_uploader/utils/importer.py

```diff
@@ -305,30 +386,20 @@ def import_samples_from_file(
         sample_file = validate_params(params)
         ws_name = params.get('workspace_name')
         df = load_file(sample_file, header_row_index, date_columns)
-        df, columns_to_input_names = format_input_file(df, {}, aliases)
-
-        # TODO: Make sure to check all possible name fields, even when not parameterized
-        if params.get('name_field'):
-            name_field = upload_key_format(params.get('name_field'))
-            if name_field not in list(df.columns):
-                raise ValueError(
-                    f"The expected name field column \"{name_field}\" could not be found. "
-                    "Adjust your parameters or input such that the following are correct:\n"
-                    f"- File Format: {params.get('file_format')} (the format to which your sample data conforms)\n"
-                    f"- ID Field: {params.get('name_field','name')}\n (the header of the column containing your names)\n"
-                    f"- Headers Row: {params.get('header_row_index')} (the row # where column headers are located in your spreadsheet)"
-                )
-            # here we rename whatever the id field was/is to "id"
-            columns_to_input_names["name"] = columns_to_input_names.pop(name_field)
-            df.rename(columns={name_field: "name"}, inplace=True)
+
+        file_format = params['file_format'].lower()
+        if 'sample_template' not in df:
+            df['sample_template'] = file_format.upper()
+
+        df, columns_to_input_names = format_input_file(df, params, {}, aliases)
 
         if not errors.get(severity='error'):
-            if params['file_format'].lower() in ['sesar', "enigma"]:
+            if file_format in ['sesar', "enigma"]:
                 if 'material' in df.columns:
-                    df.rename(columns={"material": params['file_format'].lower() + ":material"}, inplace=True)
+                    df.rename(columns={"material": file_format + ":material"}, inplace=True)
```

Comment:
I think we can get rid of this section now that we have the aliasing. we can do that in another PR though